bundler/spec/commands/newgem_spec.rb

```diff
@@ -1149,15 +1149,15 @@ def create_temporary_dir(dir)
       it_behaves_like "--no-mit flag"
     end
 
-    context "with coc option in bundle config settings set to true" do
+    context "with coc option in bundle config settings set to true", :git => ">= 2.28.0" do
```

Comment:
Can you instead tag the specific spec that runs `git config --global init.defaultBranch main` and then checks that links to `main`are present in the generated README? It t would make it more apparent why the spec only works on some git versions, and allows you to run all the other specs that do pass.

Reply:
Is this right? https://github.com/rubygems/rubygems/pull/4533/commits/691b61751e7324564394cdab4ba6398170d50080

Reply:
Ah, I missed "then checks that links to mainare present in the generated README".

Reply:
which I believe is the only spec that should be failing for you.

Reply:
Ahh, I see! https://github.com/rubygems/rubygems/pull/4533/commits/dab3fc5de156b47bd23dea2adfa5178c133a4c23 It works great on my machine.

Reply:
Yes, I meant exactly that, except that you missed the ">=" in the filter! Thanks @mame!

Reply:
Fixed 😓